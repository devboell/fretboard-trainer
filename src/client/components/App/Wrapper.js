import styled from 'styled-components'

const Wrapper = styled.div`
  width: 80%;
  margin: 10px 10%;
`

export default Wrapper
