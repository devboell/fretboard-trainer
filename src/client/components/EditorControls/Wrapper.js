import styled from 'styled-components'

const Wrapper = styled.div`
  height: 10%;
`

export default Wrapper
