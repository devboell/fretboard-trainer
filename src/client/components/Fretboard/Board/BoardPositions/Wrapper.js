import styled from 'styled-components'

const PositionWrapper = styled.div`
  display: flex;
  width: 100%;
  height: 100%;
`

export default PositionWrapper
