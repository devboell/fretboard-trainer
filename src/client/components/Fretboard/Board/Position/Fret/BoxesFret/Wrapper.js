import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  height: 80%;
  width: 90%;
  margin: 10% 5%;
  border: 1px solid darkgray;
  border-radius: 3px;
`

export default Wrapper
