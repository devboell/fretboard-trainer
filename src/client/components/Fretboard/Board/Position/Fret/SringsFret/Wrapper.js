import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0 5%;
  background-color: ${props => props.theme.background};
`

export default Wrapper
