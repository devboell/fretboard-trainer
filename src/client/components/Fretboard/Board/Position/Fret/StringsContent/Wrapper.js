import styled from 'styled-components'

const background = ({ theme, isHighlighted }) => (
  isHighlighted
    ? theme.highlight
    : theme.background
)

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 0 5%;
  background-color: ${props => background(props)};
`

export default Wrapper
