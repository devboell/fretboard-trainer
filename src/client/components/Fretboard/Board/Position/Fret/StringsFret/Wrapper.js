import styled, { css } from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  ${props => props.isHighlighted && css`
    height:100%;
    width: auto;
    -moz-border-radius: 50%; 
    -webkit-border-radius: 50%; 
    border-radius: 50%;
    background: #4679BD; 
  `}
`
//     padding-bottom: 50%;

//     background-color: ${props.theme.highlight};


export default Wrapper
