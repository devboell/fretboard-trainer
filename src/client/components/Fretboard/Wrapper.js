import styled from 'styled-components'

const Wrapper = styled.div`
  width: 100%;
  height: ${props => props.height}px;
  background-color: ${props => props.theme.background};
`

/** @component */
export default Wrapper
