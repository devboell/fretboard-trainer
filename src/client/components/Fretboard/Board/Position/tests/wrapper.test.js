import React from 'react'
import Wrapper from '../Wrapper'

const props = {
  width: 10,
}

it('should return wrapper with correct width', () => {
  const wrapper = shallow(<Wrapper {...props} />)
  expect(wrapper).toMatchSnapshot()
})
