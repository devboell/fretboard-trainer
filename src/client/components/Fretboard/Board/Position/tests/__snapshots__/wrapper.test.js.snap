// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return wrapper with correct width 1`] = `
<div
  className="sc-bdVaJa tEJaT"
  width={10}
/>
`;
